app/src/main/java/no/ntnu/klubbhuset/service/UserService.java

```diff
@@ -36,23 +49,51 @@ public JSONObject createNewUser(String firstname, String lastname, String email,
         } catch (JSONException e) {
             e.printStackTrace();
         }
+        String requestBody = newUserJson.toString();
 
-
-        JsonObjectRequest jsonObjectRequest = new JsonObjectRequest(Request.Method.POST, URL, newUserJson,
+        StringRequest jsonObjectRequest = new StringRequest(Request.Method.POST, URL,
```

Comment:
Name of variable does not reflect it's DataType.

Reply:
Burde vel streng tatt vært void, men det finnes ikke...